gulpfile.js

```diff
@@ -267,30 +266,31 @@ var reloadBrowser = function (done) {
 	done();
 };
 
+var styles = parallel(lintStyles, buildStyles);
+var scripts = parallel(lintScripts, buildScripts);
 
 // Watch for changes
-var watchSource = function (done) {
-	watch(paths.input, series(exports.default, reloadBrowser));
-	done();
+var watchSource = function() {
+	watch(paths.styles.input, styles);
+	watch(paths.scripts.input, series(scripts, reloadBrowser));
+	watch(paths.images.input, series(processImages, reloadBrowser));
+	watch(paths.icons.input, series(processIcons, reloadBrowser));
+	watch(paths.svgs.input, series(buildSVGs, reloadBrowser));
+	watch('./dist/**/*.html', reloadBrowser);
 };
```

Comment:
🔍 Although this is more verbose than the previous watch script, it makes rebuilds more incremental. Changes to `.js` files, for instance, will only trigger a script rebuild, rather than rebuilding every source type in the project

Note: We don't run `reloadBrowser` on style changes because `.stream()` sends the css to the browser as it compiles.